catalog/imaging/imaging-actionprovider-chip/src/main/java/org/codice/alliance/imaging/chip/actionprovider/ImagingChipActionProvider.java

```diff
@@ -26,22 +29,25 @@
 import ddf.action.Action;
 import ddf.action.MultiActionProvider;
 import ddf.action.impl.ActionImpl;
+import ddf.catalog.data.Attribute;
 import ddf.catalog.data.Metacard;
+import ddf.catalog.data.types.Core;
 
 public class ImagingChipActionProvider implements MultiActionProvider {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ImagingChipActionProvider.class);
 
     public static final String TITLE = "Chip Image";
 
-    public static final String DESCRIPTION =
-            "Opens a new window to enter the boundaries of an image chip for a Metacard.";
+    public static final String DESCRIPTION = "Opens a new window to enter the boundaries of an image chip for a Metacard.";
 
     public static final String PATH = "/chipping/chipping.html";
 
     public static final String ID = "catalog.data.metacard.image.chipping";
 
-    public static final String NITF_CONTENT_TYPE = "image/nitf";
+    public static final String IMAGE_NITF_METACARD_TYPE = "image";
```

Comment:
This seems like a pretty generic name for a nitf metacard type. Are we concerned we might bump up against other image types in the future - created by us or by integrators - where we might mistakenly think we can handle a metacard that doesn't contain a nitf?